x-pack/plugins/ml/public/application/components/jobs_awaiting_node_warning/jobs_awaiting_node_warning.tsx

```diff
@@ -37,7 +37,7 @@ export const JobsAwaitingNodeWarning: FC<Props> = ({ jobCount }) => {
         <div>
           <FormattedMessage
             id="xpack.ml.jobsAwaitingNodeWarning.noMLNodesAvailableDescription"
-            defaultMessage="There {jobCount, plural, one {is} other {are}} {jobCount, plural, one {# job} other {# jobs}} waiting for machine learning nodes to start."
+            defaultMessage="There {jobCount, plural, one {is} other {are}} {jobCount, plural, one {# job} other {# jobs}} waiting for autoscaling to add ML capacity. Please wait up to 5 minutes for this to happen."
```

Comment:
@droberts195 can we guarantee that this will happen within 5 minutes? Is it safer to remove the second sentence altogether?

Reply:
No, it's not guaranteed. On classic Cloud the poll of the autoscaling API is every 5 minutes, and then there's the time taken for a plan change after that, so if you're unlucky it could be 10 minutes. Serverless should react much faster.

I think it's good to give _some_ idea of timing. Maybe, "This may take several minutes." for the second sentence?

Reply:
`This may take several minutes.` for the second sentence sounds good to me.